components/Search/Search.jsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import Select from "react-select/base";
```

Comment:
It seems that react-select is redundant for this task. It's main feature is multiselect.

- a lot of unnecessary unused logic
- there is no modularity with a weight of 28 kb (https://bundlephobia.com/package/react-select@5.5.0)

Reply:
It seems that we will have a fairly large API to understand the other.

We already have not only list items inside the results window, but also groupings with headings